src/Actor.cpp

```diff
@@ -872,6 +874,14 @@ bool Actor::IsFirstUpdate() const
 void Actor::Update( float fDeltaTime )
 {
 //	LOG->Trace( "Actor::Update( %f )", fDeltaTime );
+
+	const float& rate = GameLoop::GetUpdateRate();
```

Comment:
```suggestion
	const float rate = GameLoop::GetUpdateRate();
```

Reply:
Is the reasoning here that the copy is insignificant in the grand scheme of things?

Reply:
Done. Thanks for the review and the tip.